perfkitbenchmarker/providers/gcp/util.py

```diff
@@ -26,7 +26,7 @@
 @functools32.lru_cache()
 def GetDefaultProject():
   """Get the default project."""
-  cmd = ['gcloud', 'config', 'list', '--format=json']
+  cmd = [FLAGS.gcloud_path, 'config', 'list', '--format=json']
```

Comment:
This change doesn't below to here. We need to figure out what happened.

Please find out the latest state of this file on GCP/master and then revert this one to that state.